fvm/state/accounts.go

```diff
@@ -32,17 +32,38 @@ func keyPublicKey(index uint64) string {
 	return fmt.Sprintf("public_key_%d", index)
 }
 
-type Accounts struct {
+type Accounts interface {
```

Comment:
We actually were planing last year to break this into smaller interfaces like AccountValues, AccountKeys, AccountContracts, AccountMeta, ... and Accounts to be an interface of all other interfaces.